### ISSUE TITLE
Dropdown issue

### ISSUE THREAD

**Issue by StasiekDivante** [DivanteLtd/vue-storefront#807]
Now button behaves differently when text label or arrow are pressed, whole button should show subcategories, text label should not redirect to category

<img width="246" alt="2i" src="https://user-images.githubusercontent.com/37068687/36984621-99ea0158-2095-11e8-8dd4-6ff0144626b4.png">
<img width="243" alt="3i" src="https://user-images.githubusercontent.com/37068687/36984622-9a079df8-2095-11e8-8389-526d9a41bd00.png">

**Comment by pkarw** [comment]
@StasiekDivante adter such change the user will be able to browse just the bottom level categories.
I understand your reasoning that the but I would argue which solution is better from the User point of view

Let @karolbzik decide :)

**Comment by StasiekDivante** [comment]
@pkarw together with @karolbzik we decided to choose button to show subcategories. It might be slighly worse from the user point of view, but it's a fair trade for the sake of consistency

**Comment by pkarw** [comment]
OK, so it's ready to development, hoping for finding any brave soul to fix it :)

**Comment by BlitZ-UK** [comment]
Ok, am i right in saying then that the desired functionality for this is that the far right ">" arrow should view the subcategories (in the menu) and clicking the category name should take you to the category itself?

**Comment by StasiekDivante** [comment]
hey @BlitZ-UK! That's the way it works now, arrow and text label should have the same functionality, redirecting to subcategories menu

**Comment by BlitZ-UK** [comment]
Yep that's correct, i was just double checking the functionality of how it should be - If we're removing the ability to go directly to the category, and it'll all now instead display the subcategories, can we add a "View All" button at the top of the subcategories shown, which will then take you to the category? That way you can still get to the top level category from the Menu.

**Comment by StasiekDivante** [comment]
Correct! Just be sure to keep same style, and place View All at the very top
<img width="354" alt="scr" src="https://user-images.githubusercontent.com/37068687/37032717-cedeaec8-2143-11e8-9bd4-5d34279e5ecc.png">

**Comment by BlitZ-UK** [comment]
Sounds like a plan, i'll get this implemented.

**Comment by anqaka** [comment]
@MikeSheward are you still working on this issue? Not sure if I can assign it or it already "in process" :)

**Comment by pkarw** [comment]
As far as I know @MikeSheward is now working on the Payment's related issue; so I belive you can just take this one!

**Comment by anqaka** [comment]
Ok, I'll try to implement this

**Comment by Januszpl** [comment]
@pkarw I think this one should be closed
